data_model/src/query.rs

```diff
@@ -989,8 +993,15 @@ pub mod asset {
 
     impl FindAllAssets {
         /// Construct [`FindAllAssets`].
-        pub const fn new() -> Self {
-            FindAllAssets
+        pub fn new() -> Self {
+            Self { sort_by_key: None }
+        }
+
+        /// Construct [`FindAllAssets`] with sort by key.
+        #[must_use]
+        pub fn with_sort(mut self, sort_by_key: impl Into<EvaluatesTo<Name>>) -> Self {
```

Comment:
I think that just one constructor would be enough

Reply:
Don't mind having two. 

`#[inline]` both.

Reply:
I think no because, for a large number of cases, clients will have to specify disabled sorting like this:
```rust
FindAllAssets::new(None)
```